dtfs-central-api/src/services/state-machines/utilisation-report/event-handlers/helpers/calculate-initial-utilisation-and-fixed-fee.test.ts

```diff
@@ -0,0 +1,89 @@
+import { calculateInitialUtilisation } from '@ukef/dtfs2-common';
+import { calculateInitialUtilisationAndFixedFee, hasRequiredValues } from './calculate-initial-utilisation-and-fixed-fee';
+import { TfmFacilitiesRepo } from '../../../../../repositories/tfm-facilities-repo';
+import { aTfmFacility } from '../../../../../../test-helpers';
+import { calculateFixedFee } from './calculate-fixed-fee';
+
+describe('helpers/calculate-initial-utilisation-and-fixed-fee', () => {
+  describe('hasRequiredValues', () => {
+    it('should return true if all values are present', () => {
+      const result = hasRequiredValues(1, 2, 3, new Date(), new Date());
+      expect(result).toEqual(true);
+    });
+
+    it('should return false if a value is missing', () => {
+      const result = hasRequiredValues(1, 2, 3, new Date(), null);
+      expect(result).toEqual(false);
+    });
+
+    it('should return false if a date is the wrong format', () => {
+      const result = hasRequiredValues(1, 2, 3, 'a', new Date());
```

Comment:
Should we have `hasRequiredValues` with object structured params so it's less ambiguous when calling?

```ts
hasRequiredValues({
  value: 1,
  interestPercentage: 2,
  dayCountBasis: 3,
  coverStartDate: 'a',
  coverEndDate: new Date(),
});
```